VotingApplication/VotingApplication.Web/Scripts/Services/RoutingService.js

```diff
@@ -99,3 +99,105 @@
         }
     }
 })();
+
```

Comment:
Again, I get that it's a copy/paste, but should we be using this as an opportunity to clean up? In which case we should only be copying over functions as and when we need them, rather than wholesale, which could leave us with unused ones.